variables.tf

```diff
@@ -1,9 +1,228 @@
+  default     = false
+}
+variable "reloader_ignore_configmaps" {
+  description = "Whether to ignore configmap changes or not"
+  type        = bool
+  default     = false
+}
+variable "reloader_is_openshift" {
+  description = "Enable OpenShift DeploymentConfigs"
+  type        = bool
+  default     = true
+}
+variable "reloader_is_argo_rollouts" {
+  description = "Enable Argo Rollouts"
+  type        = bool
+  default     = false
+}
+variable "reloader_reload_on_create" {
+  description = "Enable reload on create events"
+  type        = bool
+  default     = true
+
+}
+variable "reloader_sync_after_restart" {
+  description = "Enable sync after Reloader restarts for Add events, works only when reloadOnCreate is true"
+  type        = bool
+  default     = true
+
+}
+variable "reloader_pod_monitor_metrics" {
+  description = "Enable to scrape Reloader's Prometheus metrics"
+  type        = bool
+  default     = false
+}
+
+variable "reloader_log_format" {
+  description = "The log format to use for reloader. Possible values are `json` or `text`. Default value is `json`"
+  type        = string
+  default     = "text"
+  validation {
+    condition     = contains(["json", "text"], var.reloader_log_format)
+    error_message = "The specified reloader_log_format is not a valid selection! Valid values are `json` or `text`"
+  }
+}
+variable "reloader_custom_values" {
+  description = "String containing custom values to be used for reloader helm chart. See https://github.com/stakater/Reloader/blob/master/deployments/kubernetes/chart/reloader/values.yaml"
+  type        = string
+  default     = null
+}
+
+# reloader image and helm charts references
+
+variable "reloader_image" {
+  type        = string
+  description = "The reloader image in the format of `[registry-url]/[namespace]/[image]`."
+  default     = "ghcr.io/stakater/reloader"
+  nullable    = false
+}
+
+variable "reloader_image_version" {
+  type        = string
+  description = "The version or digest for the reloader image to deploy."
+  default     = "v1.2.1-ubi@sha256:80a557100c6835c7e3c9842194250c9c4ca78f43200bc3a93a32e5b105ad11bb" # datasource: ghcr.io/stakater/reloader
+  nullable    = false
+  validation {
+    condition     = can(regex("(^v\\d+\\.\\d+.\\d+(\\-\\w+)?(\\@sha256\\:\\w+){0,1})$", var.reloader_image_version))
+    error_message = "The value of the reloader image version must match classic version or the tag and sha256 image digest format"
+  }
+}
+
+variable "reloader_chart_location" {
+  type        = string
+  description = "The location of the Reloader Helm chart."
+  default     = "https://stakater.github.io/stakater-charts"
+  nullable    = false
+}
+
+variable "reloader_chart_version" {
+  type        = string
+  description = "The version of the Reloader Helm chart. Ensure that the chart version is compatible with the image version specified in reloader_image_version."
+  # renovate: datasource=github-releases depName=stakater/Reloader
+  default  = "1.2.0"
```

Comment:
This version is out of sync with the default image version specified in the `reloader_image_version` input. Can you ensure they match please